kombu/transport/kafka.py

```diff
@@ -66,30 +124,32 @@ def fetch_offsets(self, client, topic, offset):
         :type client:  :class:`pykafka.KafkaClient`
         :param topic:  Name of the topic.
         :type topic:  :class:`pykafka.topic.Topic`
-        :param offset: Offset to reset to. Can be earliest, latest or a datetime.
-            Using a datetime will reset the offset to the latest message published
-            *before* the datetime.
+        :param offset: Offset to reset to. Can be earliest, latest or a
+            datetime.  Using a datetime will reset the offset to the latest
+            message published *before* the datetime.
         :type offset: :class:`pykafka.common.OffsetType` or
             :class:`datetime.datetime`
-        :returns: {partition_id: :class:`pykafka.protocol.OffsetPartitionResponse`}
+        :returns: {
+                partition_id: :class:`pykafka.protocol.OffsetPartitionResponse`
+            }
         """
         if offset.lower() == 'earliest':
             return topic.earliest_available_offsets()
         elif offset.lower() == 'latest':
             return topic.latest_available_offsets()
         else:
-            offset = dt.datetime.strptime(offset, "%Y-%m-%dT%H:%M:%S")
+            offset = datetime.datetime.strptime(offset, "%Y-%m-%dT%H:%M:%S")
             offset = int(calendar.timegm(offset.utctimetuple())*1000)
             return topic.fetch_offset_limits(offset)
 
-
-    def sanatize_queue_name(self, queue):
-        """Need to sanatize the queue name, celery sometimes pushes in @ signs"""
+    def sanitize_queue_name(self, queue):
+        """Need to sanitize the queue name, celery sometimes pushes in @
+        signs"""
         return str(queue).replace('@', '')
```

Comment:
@erickt @BenjaminDavison  This will actually break the celery control API. Where queues like `celery@<hostname>.celery.pidbox` get created on the fly and gets used for all control signal handling(like `stats` or `cancel_consumer` etc..). Any thoughts ?